nemoguardrails/library/prompt_security/flows.v1.co

```diff
@@ -0,0 +1,22 @@
+# INPUT RAILS
+
+define subflow protect prompt
+  """Check if the prompt is valid according to Prompt Security."""
+  $result = execute protect_text
```

Comment:
Per suggestion in this [comment](https://github.com/NVIDIA/NeMo-Guardrails/pull/920#issuecomment-2586737364)

```suggestion
  $result = execute protect_text(user_prompt=$user_message)
```